paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsActivityTest.kt

```diff
@@ -355,53 +331,22 @@ internal class PaymentOptionsActivityTest {
         assertThat(scenario.state).isEqualTo(Lifecycle.State.DESTROYED)
     }
 
-    @Test
-    fun `Clears error on user selection`() {
```

Comment:
From what I can tell, this can’t really happen in `PaymentOptionsActivity`. It’s more of a `PaymentSheetActivity` scenario, because we execute network calls in that one.